docs/development/index.md

```diff
@@ -12,4 +12,47 @@ Fluidd is built using VueJS, and the Vuetify Framework to provide a cohesive,
 easily to implement UI.
 {: .fs-6 .fw-300 }
 
-Further details on development environments will come at a later date.
+## Running Fluidd locally
+
+### Install Node v14
+
+Follow the instructions from [NodeSource](https://github.com/nodesource/distributions/blob/master/README.md#deb)
```

Comment:
I don't think recommending some third-party installer is the best
https://nodejs.org/en/ is the official one

Reply:
NodeSource is the official source of binary distributions for Node.js installed on Linux distros via a package manager:

https://nodejs.org/en/download/package-manager/#debian-and-ubuntu-based-linux-distributions

Reply:
Ah intersting, but it's for "Debian and Ubuntu based Linux distributions"

So I'd still prefer linking to the official site, not everyone uses debian/ubuntu or even linux